faker.js

```diff
@@ -1,39 +1,59 @@
 const faker = require('faker');
-const db = require('./database/index.js');
+const postgres = require('./database/index.js');
+const cassandra = require('./database/cassandra.js');
 
-const insertIntoDb = (numOfTimes = 103) => {
-  var users = [];
-  for (let i = 0; i < numOfTimes; i++) {
-    const user = {
-      carId: faker.random.number({
+const insertIntoDb = (numOfTimes = 2000) => {
+  const reviews = [];
+  for (let i = 0; i < numOfTimes; i += 1) {
+    const review = {
+      carid: faker.random.number({
         min: 1,
-        max: 30,
+        max: 100,
       }),
-      userName: faker.name.findName(),
-      userReview: faker.lorem.sentences(3),
-      userReviewDate: faker.date.recent(5).toString().slice(4, 15),
-      userRating: faker.random.number({
+      name: faker.name.findName(),
+      review: faker.lorem.sentences(3),
+      rating: faker.random.number({
         min: 1,
         max: 5,
-      })
-    }
-    users.push(user);
-
+      }),
+      date: faker.date.recent(5).toString().slice(4, 15),
+    };
+    reviews.push(review);
   }
 
-  users.sort(function compare(a, b) {
-    if (a[4] < b[4]) {
-      return -1;
-    }
-    if (a[4] > b[4]) {
-      return 1;
-    }
-    return 0;
-  });
+  // reviews.sort(function compare(a, b) {
+  //   if (a[4] < b[4]) {
+  //     return -1;
+  //   }
+  //   if (a[4] > b[4]) {
+  //     return 1;
+  //   }
+  //   return 0;
+  // });
 
-  users.forEach( (currentElement) => {
-    db.addNewUser(currentElement);
-  })
+  let casTimer = 150;
+  let index = 0;
+  while (index < 10000) {
+    setTimeout(() => {
+      reviews.forEach((review) => {
+        cassandra.execute(`INSERT INTO reviews.reviews(id, carid, name, review, rating, date) VALUES (uuid(), ${review.carid}, $$${review.name}$$, $$${review.review}$$, ${review.rating}, $$${review.date}$$)`, (err, result) => {
```

Comment:
Are you doing a DB operation for each of the 10M entries? Or are you batching somehow? Can't quite tell. If you are doing individual -- is it faster to batch?